test-schemas/src/main/resources/openapi/simple.yaml

```diff
@@ -97,6 +97,11 @@ components:
   schemas:
     dto:
       type: object
+      x-obfuscated:
```

Comment:
I believe it's better to move this property at value-type level.
Also this feature is very specific to languages like java with method `toString`. For example, it's not applicable to TypeScript.

Maybe it's better to implement this feature like this

```yaml
type: object:
properties:
  property1:
    type: string
    x-to-string-obfuscated: true
```
```

Reply:
1) i tried to implement both ways (class-type and value-type), as for required, but had no success (no kotlin experience before);
i agree to create debt on it

2) name was getting from https://github.com/swagger-api/swagger-codegen/pull/7060 ; 
don't think its good idea to create another alias for this feature

Reply:
ok, it's better to implement it as it's already implemented in swagger-codegen

Reply:
first comment is fixed